tox.ini

```diff
@@ -2,7 +2,7 @@
 envlist = py26, py27, py33, py34, pypy, flake8
 
 [testenv]
-commands = py.test --cov cookiecutter tests {posargs}
+commands = py.test --cov cookiecutter {posargs}
```

Comment:
I usually use `{posargs:tests}`. This way by default, it will only look in the `tests` directory but the user can override and specify specific tests and things won't be run twice.